ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/execution/ExecutionPayload.java

```diff
@@ -103,6 +103,11 @@ public ExecutionPayload(
         transactions);
   }
 
+  @Override
+  public boolean isDefault() {
+    return super.isDefault();
+  }
```

Comment:
Can probably just omit this since it's just calling the super implementation anyway.

Reply:
intellij was insistent but i'll try again...